### PR TITLE
Pull Request - SOC Process Framework Workbook and Get-SOCActions Playbook

### PR DESCRIPTION
**Pull request by rinure-msft**
Fixes #

## Proposed Changes

  - Added Workbook - SOC Process Framework v1.0
  - Uploaded associated Images for Workbook
  - Updated WorkbookMetadata file
  - Added Get-SOCActions Playbook
  - Worked with Clive Watson to add SocRA Watchlist and update Incident Overview Workbook for commit

### CODE REVIEW

**Comment by rinure-msft** [comment]
Please let me know if there is anything else I need to do on my end. The community is looking forward to this content! It has been widely discussed and customers, partners, and internal field folk are all looking forward to leveraging this content to help increase the adoption of Azure Sentinel.

**Comment by sarah-yo** [comment]
Hey @rinure-msft a few of the checks are failing - documents link is just taking out the localization stuff "en-us" from all your URLs.

Here's your error for the workbooks validation check - _WorkbooksTemplate Validation Failed. File path: Workbooks/SOCProcessFramework.json. Error message: Contains info of a resource at offset 1025. A workbook template must not contain any references to resources.__

**Comment by rinure-msft** [comment]
Hi Sarah,

This was/is my first time submitting a workbook, so I am unfamiliar with the checks that are failing. Can you please help me understand what I need to do to address these issues? I want to make this available, but realize I may need help addressing the checks.

Kind regards,
Rin

File: Workbooks/SOCProcessFramework.json [hunk 1]
```diff
@@ -0,0 +1,1521 @@
...
+                  "linkTarget": "parameter",
+                  "linkLabel": "Shift Turnover",
+                  "subTarget": "Shift Turnover",
+                  "style": "secondary"
+                },
+                {
+                  "id": "8eb257d2-b83a-462a-b5d7-f356752edb9d",
+                  "cellValue": "Tab",
+                  "linkTarget": "parameter",
+                  "linkLabel": "Weekly Oerational Process",
```
Comment (sarah-yo) [inline Workbooks/SOCProcessFramework.json:1027]: typo - "operational"

**Pull request merged**